app/queries/CommodityUpdateQuery.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package queries
+
+import models.Commodity
+import play.api.libs.json.JsPath
+
+object CommodityUpdateQuery extends Gettable[Commodity] with Settable[Commodity] {
+
+  override def path: JsPath = JsPath \ "commodity"
```

Comment:
Will want to give it another name here. This is the JSON name in the User Answers and its shared with the original CommodityQuery. So if you use "commodity" here it will overwrite the CommodityQuery data and they will start reading each others data etc

When you add the recordId you'll want to stick that in the path anyway, so would probably avoid overwrite, but I'd give it another name for niceness sake anyway :)

Reply:
commodityUpdate or may be commdityCodeUpdate ?